huaweicloud/config/config.go

```diff
@@ -104,38 +69,22 @@ type Config struct {
 	// SecurityKeyLock is used to make the accessing of SecurityKeyExpiresAt serial,
 	// prevent sending duplicate query metadata api
 	SecurityKeyLock *sync.Mutex
+
+	// Legacy
+	Username         string
+	UserID           string
+	Password         string
+	AgencyName       string
+	AgencyDomainName string
+	DelegatedProject string
 }
 
 func (c *Config) LoadAndValidate() error {
 	if c.MaxRetries < 0 {
-		return fmt.Errorf("max_retries should be a positive value")
+		return fmt.Errorf("max_retries should be a positive value.")
```

Comment:
go-lint: error strings should not be capitalized or end with punctuation or a newline

Reply:
done